CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 # Unreleased
 
-Nothing Yet!
+This release is mostly several fixes to how we create GitHub Releases, as well as some internal improvements for future feature work.
+
+
+## GitHub Release Ordering
+
+We now prefer creating your [GitHub Release in the host step](https://opensource.axo.dev/cargo-dist/book/reference/config.html#github-release), ensuring published npm and Homebrew packages never refer to URLs that don't yet exist.
+
+* [docs](https://opensource.axo.dev/cargo-dist/book/reference/config.html#github-release)
+* @mistydemeo [fix: prefer creating github releases in host step](https://github.com/axodotdev/cargo-dist/pull/1171)
+
+
+## GitHub Release Reliability
+
+GitHub Releases should once again be created transactionally, preventing a release from being created without its artifacts being uploaded. This fixes a regression from the previous release.
```

Comment:
we may want to be more specific here about when the regression happened as it wasn't unilateral

Reply:
not sure I follow what detail you're thinking of